specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json

```diff
@@ -5708,6 +5772,29 @@
         }
       }
     },
+    "BudgetInformation": {
+      "description": "Budget information.",
+      "properties": {
+        "amount": {
+          "description": "The budget amount.",
+          "$ref": "#/definitions/Amount"
+        },
+        "startDate": {
+          "description": "Start Date.",
+          "type": "string",
+          "format": "date-time"
+        },
+        "endDate": {
+          "description": "End Date.",
+          "type": "string",
+          "format": "date-time"
+        },
+        "additionalDetails": {
+          "description": "This is an free flow string.",
```

Comment:
s/an/a

Reply:
Any reason we don't make this an object, so someone can assign an arbitrary object instead? E.g.:

```
"additionalDetails": {
  "taskOrderId": "123",
  "x": 42
}